#request
both_arm_trajectory trajectory
---
#response 
bool success
